debian/ellis.init.d

```diff
@@ -124,26 +92,18 @@ do_start()
 #
 do_stop()
 {
-	# Return
-	#   0 if daemon has been stopped
-	#   1 if daemon was already stopped
-	#   2 if daemon could not be stopped
-	#   other if a failure occurred
-
-        # Stopping Ellis terminates all processes that use its version of
-        # python, including any of its tools. Wait for the tools to complete,
-        # otherwise we can break automated installs.
-        wait_for_tools || return 2
```

Comment:
Why have we removed this?  Have we tested that automated installs are still reliable?  ...and do we know which automated installs were normally broken, so that we can focus our testing on that?